corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -263,7 +263,7 @@
                 <div class="accordion-body collapse 
                     {% if cases|length == 1 %}in{% endif %}" 
                     id="form-case-acc-{{ forloop.counter }}">
-                    <div class="accordion-inner">
+                    <div class="accordion-inner" style="display: inline-block">
```

Comment:
This seems like a brand-aid fix, but in spite of 1-2 hours I couldn't find the reason this broke in the first place. Nothing seems to have changed relating to styles here.